vnext/local-cli/runWindows/utils/build.js

```diff
@@ -25,6 +25,7 @@ async function buildSolution(
   msBuildProps,
   verbose,
   target,
+  buildLogDirectory,
```

Comment:
Consider having buildSolution take in an object literal instead of many params.

Reply:
I'll send a separate Pr for that after this one, to keep the changes small. Note the file is not TS, nor flow so there will not be any checking for args.